src/pokemanki/gui/pokemanki_trade.py

```diff
@@ -0,0 +1,197 @@
+# Form implementation generated from reading ui file 'designer/pokemanki_trade.ui'
+#
+# Created by: PyQt6 UI code generator 6.4.0
+#
+# WARNING: Any manual changes made to this file will be lost when pyuic6 is
+# run again.  Do not edit this file unless you know what you are doing.
+
+import ctypes
+import platform
+
+from aqt import QDialog
+from aqt.webview import AnkiWebView
+from PyQt6 import QtCore, QtGui, QtWidgets
+
+
+class TradeWindow(object):
+    dialog = None
+    webView = None
+
+    def __init__(self, mw, parent):
+        self.dialog = QDialog()
+        self.dialog.setObjectName("Dialog")
+        self.dialog.resize(400, 472)
+        self.dialog.setMinimumSize(QtCore.QSize(250, 100))
+        self.verticalLayout = QtWidgets.QVBoxLayout(self.dialog)
+        self.verticalLayout.setSizeConstraint(
+            QtWidgets.QLayout.SizeConstraint.SetMaximumSize
+        )
+        self.verticalLayout.setSpacing(9)
+        self.verticalLayout.setObjectName("verticalLayout")
+        self.title = QtWidgets.QLabel(self.dialog)
+        self.title.setMaximumSize(QtCore.QSize(16777215, 123))
+        font = QtGui.QFont()
+        font.setPointSize(20)
+        font.setBold(True)
+        self.title.setFont(font)
+        self.title.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
+        self.title.setWordWrap(True)
+        self.title.setObjectName("title")
+        self.verticalLayout.addWidget(self.title, 0, QtCore.Qt.AlignmentFlag.AlignTop)
+
+        self.webView = AnkiWebView(parent=self.dialog, title="pokémanki trades")
```

Comment:
Since we changed `self.dialog` to the `Ui_Dialog` class from the autogenerated code, we need to pass `self` as parent for AnkiWebView to meet the required type.
Also pls use title case for titles (though I'm not sure if this is even displayed).
```suggestion
        self.webView = AnkiWebView(parent=self, title="Pokémanki Trades")
```

Reply:
The title here is the one displayed when inspecting the webViews, which Anki uses in lowercase.